deploy/components/course-calendar/course-calendar.js

```diff
@@ -980,6 +980,171 @@ var CourseCalendar = (function () {
+            // a new row
+            var row = [];
+
+            // hour cell
+            var firstCell = {
+                content: (String(hour)+":30").split("").reverse().join(""), // string reverse because of RTL problems
+                styles: {
+                    minCellHeight: 14
+                }
+            }
+            
+            row.push(firstCell);
+
+            for(var day = 0; day < 5; ++day) {
+
+                // if the hour is already filled with an event (rowspan > 1), do not create the cell
+                if(hour in hoursUsed[day])
+                    continue;
+
+                var cell = {
+                    content: "",
+                    styles: {
+                        valign: "middle",
+                        textColor: "white",
+                        fontSize: 8
+                    }
+                }
+
+                var eventKey = String(day) + "-" + String(Math.floor(hour));
+
+                // if there is an event at this hour
+                if(eventKey in events) {
+
+                    cell["content"] = events[eventKey].title;
+                    cell["rowSpan"] = events[eventKey].end-events[eventKey].start;
+                    cell["styles"]["fillColor"] = events[eventKey].bgcolor;
+
+                    // remember the used hours
+                    for(var i=events[eventKey].start; i<events[eventKey].end; ++i)
+                        hoursUsed[day][i] = true;
+                }
+
+                row.push(cell);
+            }
+
+            tableRows.push(row);
+        }
+
+
+        /* PDF Generation */
+
+        // initialize jsPDF object with a landscape orientation
+        var pdf = new jsPDF({
+            orientation: "landscape"
+        })
+
+        // support for Hebrew
+        pdf.setR2L(true);
+        pdf.addFileToVFS('Rubik-normal.ttf', fontNormal);
+        pdf.addFont('Rubik-normal.ttf', 'Rubik', 'normal');
+        pdf.addFileToVFS('Rubik-bold.ttf', fontBold);
+        pdf.addFont('Rubik-bold.ttf', 'Rubik', 'bold');
+        pdf.setFont('Rubik');
+
+        // logo
+        pdf.setR2L(false);
+        pdf.setFontSize(8);
+        pdf.text("CheeseFork", 5, 8);
+        pdf.setR2L(true);
+        pdf.addImage(logo, 'PNG', 21, 5, 15, 6);
+
+        // title
+        pdf.setFontSize(25);
+        pdf.myText(title, 100, 15, {
+            align: "center"
+        });
+
+        // subtitle
+        pdf.setFontSize(12);
+        pdf.myText(subtitle, 100, 30, {
+            align: "center"
+        });
+
+        // table
+        pdf.autoTable({
+            head: [tableHeaders.reverse()], // reverse because of RTL
+            body: tableRows.map(function(row) {return row.reverse()}), // reverse because of RTL
+            theme: 'grid',
+            startY: 40,
+            styles: {
+                font: "Rubik",
+                halign: "center",
+                lineWidth: 0.1,
+            },
+        });
+    
+        pdf.save("Calendar.pdf");
```

Comment:
We can use a more "friendly" file name for the PDF, as we do now for the ics file (e.g. `winter-2020-2021.pdf`).